lib/puppet/feature/base.rb

```diff
@@ -32,11 +32,11 @@
     require 'win32/service'
     true
   rescue LoadError => err
-    warn "Cannot run on Microsoft Windows without the win32-process, win32-dir and win32-service gems: #{err}" unless Puppet.features.posix?
+    warn _("Cannot run on Microsoft Windows without the win32-process, win32-dir and win32-service gems: %{err}") % { err: err } unless Puppet.features.posix?
```

Comment:
Would we translate gem names - e.g. "win32-process", "win32-dir", and "win32-service"? (If not, might possibly be worth a translator note?)